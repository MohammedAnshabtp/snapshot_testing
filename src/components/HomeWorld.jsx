import React from 'react'

function HomeWorld({name}) {
  return (
    <div>HomeWorld,{name}</div>
  )
}

export default HomeWorld