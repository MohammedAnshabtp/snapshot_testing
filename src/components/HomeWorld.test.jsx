import {render} from '@testing-library/react';
import HomeWorld from '../../src/components/HomeWorld';
import {it,expect} from 'vitest';

it('render correctly',() =>{
    const{asFragment} = render(<HomeWorld name="World"/>);
    expect(asFragment()).toMatchSnapshot()
});