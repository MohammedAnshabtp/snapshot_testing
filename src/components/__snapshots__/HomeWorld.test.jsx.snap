// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`render correctly 1`] = `
<DocumentFragment>
  <div>
    HomeWorld,World
  </div>
</DocumentFragment>
`;
